uperf-wrapper/uperf-wrapper.py

```diff
@@ -133,6 +144,19 @@ def _summarize_data(data):
                              np.median(op_result),
                              np.average(op_result),
                              np.percentile(op_result, 95)))
+
+    print("")
+    print("UPerf Latecny results (microseconds):")
```

Comment:
we're multiplying by 1000 on L40, wanted to confirm if the timestamp being returned was in milliseconds. Also a minor nit with the s/Latecny/Latency :)

Reply:
yes, timestamp was in milliseconds